apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp

```diff
@@ -383,25 +383,28 @@ static void utc_media_MediaPlayer_stop_n(void)
 
 static void utc_media_MediaPlayer_getVolume_p(void)
 {
-	uint8_t volume;
+	uint8_t volume, prev;
```

Comment:
Coding guidelines violation
Please move to next line

Reply:
Fixed it